modules/fundamental/src/sbom/model/details.rs

```diff
@@ -74,6 +80,35 @@ impl SbomDetails {
             .all(tx)
             .await?;
 
+        let product_advisory_info = sbom
+            .find_related(product_version::Entity)
+            .join(JoinType::LeftJoin, product_version::Relation::Product.def())
+            .join(JoinType::LeftJoin, product::Relation::Cpe.def())
+            .join(
+                JoinType::Join,
+                trustify_entity::cpe::Relation::ProductStatus.def(),
+            )
+            .join(JoinType::Join, product_status::Relation::Status.def())
+            .join(JoinType::Join, product_status::Relation::Advisory.def())
+            .distinct_on([
+                (product_status::Entity, product_status::Column::ContextCpeId),
+                (product_status::Entity, product_status::Column::StatusId),
+                (product_status::Entity, product_status::Column::Component),
+                (
+                    product_status::Entity,
+                    product_status::Column::VulnerabilityId,
+                ),
+            ])
```

Comment:
All these joins feel like a smell to me, but I got no better suggestion.

Reply:
If you look existing search for purls, it's the similar join-armargedon :)

https://github.com/trustification/trustify/blob/main/modules/fundamental/src/sbom/model/details.rs#L42

There's A LOT of room for improvement, but we need to start somewhere.